sympy/combinatorics/tests/test_homomorphisms.py

```diff
@@ -87,6 +87,8 @@ def test_find_homomorphism():
     # Find all possible epimorphisms.
     list_hom = find_homomorphism(G, H, surjective=True, all=True)
     assert all(elem.is_surjective() for elem in list_hom)
+    check = find_homomorphism(G, H, injective=True, surjective=True, compute=False)
```

Comment:
This is the same as `is_isomorphic` which works differently from purely isomorphic cases. It would be better to have a test where `G` and `H` aren't isomorphic and we are looking for a proper injection